component/core/src/main/java/org/exoplatform/social/core/space/impl/SpaceServiceImpl.java

```diff
@@ -1759,4 +1778,27 @@ private String checkSpaceEditorPermissions(Space space) {
     }
     return editor;
   }
+
+  private List<SpaceExternalInvitation> convertSpaceExternalInvitationEntity(List<SpaceExternalInvitationEntity> spaceExternalInvitationEntities) {
```

Comment:
you can call the conversion methods fromSpaceExternalInvitationEntity/toSpaceExternalInvitationEntity